css/src/mixins/font-size.scss

```diff
@@ -0,0 +1,39 @@
+@use 'sass:math';
+
+/*
+ * A note on how to test changes made to this function.
+ * 1. Uncomment the @debug lines to see your changes applied during build.
```

Comment:
Remove the whole paragraph? I don't see any `debug` lines anymore

Reply:
Removed!